src/index.js

```diff
@@ -429,6 +429,21 @@ f.analogWrite = function(pin, value, freq, callback) {
 
     // Make sure the pin has a PWM associated
     if(typeof pin.pwm == 'undefined') {
+          if(typeof pin.gpio != 'undefined' ){
+            if(value>=0.5)
+            {
+                resp=f.digitalWrite(pin,g.HIGH,callback);
+                if(callback) callback(resp);
+                return(true);
+            }
+            else
+            {
+                resp=f.digitalWrite(pin,g.LOW,callback);
+                if(callback) callback(resp);
+                return(true);
+            }
+
```

Comment:
Please don't add blank lines without a purpose.

Reply:
Sorry, this was a mistake won't happen again

Reply:
No worries. Just using this as an early opportunity to discuss style.

RIght now, it looks like there is a tabs issue. I believe the files are using 4 spaces for tabs. Can you double-check your tabs vs. spaces in the pull request?